p2p.go

```diff
@@ -381,7 +391,7 @@ func NewHost(ctx context.Context, options ...Option) (*Host, error) {
 	if err != nil {
 		return nil, err
 	}
-	ps, err := newPubSub(ctx, host, pubsub.WithBlacklist(blacklist))
+	ps, err := newPubSub(ctx, host, pubsub.WithBlacklist(blacklist), pubsub.WithMaxMessageSize(cfg.MaxMessageSize))
```

Comment:
this is the change that fix the issue (> 1M byte block) last night?

Reply:
yes，after updating config, the 1000tps test passed